lib/st/api/lims.pm

```diff
@@ -120,6 +120,12 @@ has 'driver_type' => (
                      );
 sub _build_driver_type {
   my $self = shift;
+  if($self->has_driver){
```

Comment:
1. Changes file update?
2. Driver attribute is of unknown type, it might be possible to set it to anything. Not clear why... Has to be fixed at some point.
Might be safer to have the following
if($self->has_driver && $self->driver) {
my $type = ref $self->driver;
if ($type) {
    my $prefix = __PACKAGE__ . q(::);
    $type =~ s/\A\Q$prefix\E//smx;
    return $type;
  }
}